spec/services/subscription_validator_spec.rb

```diff
@@ -357,19 +356,19 @@ def stub_validations(validator, methods)
           end
 
           context "and the customer is associated with a user" do
-            let(:customer) { instance_double(Customer, user: user) }
+            before { expect(customer).to receive(:user).once { user } }
```

Comment:
this is should be an `allow` ⚠️

Reply:
Can you explain why using an `expect` is a problem? Is it just because we are testing the same thing for each example?

Reply:
It is just that you're stubbing a method here and not asserting anything, like `expect` is meant for.

Reply:
What makes you think I'm not asserting something? This expect will still cause the spec to fail if the assertion is not met.